metron-platform/metron-data-management/src/main/java/org/apache/metron/dataloads/extractor/stix/StixExtractor.java

```diff
@@ -38,6 +39,7 @@
 import java.util.Map;
 
 public class StixExtractor implements Extractor {
+    private static final Logger LOG = Logger.getLogger(StixExtractor.class);
```

Comment:
Yeah, this ended up starting live as a patch against an older branch that I just made current.  I changed the logging to use slf4j now.  Good eye, @simonellistonball